src/schema/queries/index.js

```diff
@@ -19,15 +19,32 @@ const typeDefs = gql`
 
       processings: [IngredientProcessing]!
       processing(id: ID!): IngredientProcessing
+      processingsOfIngredient(id: ID!): [IngredientProcessing]!
```

Comment:
This we can get from `ingredients` query itself, extra query for this defeats the purpose of graphql.